app/code/Magento/CatalogImportExport/Model/Import/Product/Type/AbstractType.php

```diff
@@ -537,7 +544,11 @@ public function clearEmptyData(array $rowData)
             if (!$attrParams['is_static'] && empty($rowData[$attrCode])) {
                 unset($rowData[$attrCode]);
             }
+            if (isset($rowData[$attrCode]) && $rowData[$attrCode] === self::EMPTY_VALUE) {
+                $rowData[$attrCode] = '';
```

Comment:
I assumption here is that this will set the value to '' or 0 rather than removing the value from the DB. I think that this is ok, but I was wondering if maybe it might be also an option to clean up the database values as well. The value of 0 on an integer is different from not set and I would be worried that this could show up as an odd bug in someone's import. What are your thoughts here?

Reply:
@dmanners, I think you are right about this. The value 0 and not set for integer is it really big difference. It is also difference for varchar or text if it will be '' or null in db. Seems like it will be better to use instead $rowData[$attrCode] = '';  - $rowData[$attrCode] = null; How do you think ?

Reply:
If null cleans the values from the database then that should be good. I am not sure if/how that part of the import is setup though.

Reply:
@dmanners, yes,  it will be null. I fixed it and pr desription.